### PR TITLE
chore: improve calling logs

### PR DESCRIPTION
**Pull request by ohassine**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Description

This PR aims to improve calling logs making it more informative

Needs releases with:

- [ ] GitHub link to other pull request

#### How to Test

N/A
----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: calling/src/androidMain/kotlin/com/wire/kalium/calling/Calling.kt [hunk 1]
```diff
@@ -59,8 +59,6 @@ interface Calling : Library {
 
     fun wcall_end(inst: Handle, conversationId: String)
 
-    fun wcall_close()
```
Comment (vitorhugods) [inline calling/src/androidMain/kotlin/com/wire/kalium/calling/Calling.kt:62]: Is this intended?

Comment (ohassine) [reply calling/src/androidMain/kotlin/com/wire/kalium/calling/Calling.kt:62]: yes, It's not used

File: logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/scenario/OnAnsweredCall.kt [hunk 2]
```diff
@@ -4,18 +4,18 @@ import com.sun.jna.Pointer
...
-import com.wire.kalium.logic.feature.call.CallManagerImpl
 import com.wire.kalium.logic.feature.call.CallStatus
 
 //TODO(testing): create unit test
 class OnAnsweredCall(
     private val callRepository: CallRepository
 ) : AnsweredCallHandler {
     override fun onAnsweredCall(conversationId: String, arg: Pointer?) {
-        callingLogger.i("${CallManagerImpl.TAG} -> answeredCallHandler")
+        callingLogger.i("OnAnsweredCall -> answeredCallHandler called")
```
Comment (vitorhugods) [inline logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/scenario/OnAnsweredCall.kt:14]: Here too.

**Review by vitorhugods — CHANGES_REQUESTED** [review]
Always good to have some more logs, but can we add some parameters of the function being executed in them too?

**Review by vitorhugods — APPROVED** [review]
👏🏼

**Pull request merged**